code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,983 @@
-			C.show_message("<span class='danger'>The world around you suddenly becomes quiet.</span>", 3)
-			affected++
-		if(affected)
-			usr.whisper("Sti[pick("'","`")] kaliedir!")
-			usr << "<span class='danger'>Your talisman turns into gray dust, deafening everyone around.</span>"
-			for (var/mob/V in orange(1,src))
-				if(!(iscultist(V)))
-					V.show_message("<span class='danger'>Dust flows from [usr]'s hands for a moment, and the world suddenly becomes quiet..</span>", 3)
-	return
 
 
-///Blind: Makes all non-cultists nearby blind. Combine with Deafen to make the entire shuttle go into a panic and laugh as they bumble around.
+//Rite of the Corporeal Shield: When invoked, becomes solid and cannot be passed. Invoke again to undo.
+/obj/effect/rune/wall
+	cultist_name = "Rite of the Corporeal Shield"
+	cultist_desc = "When invoked, makes the rune block passage. Can be invoked again to reverse this."
+	invocation = "Khari'd! Eske'te tannin!"
+	icon_state = "1"
+	grammar = "mgar ire ego"
+
+/obj/effect/rune/wall/examine(mob/user)
+	..()
+	if(density)
+		user << "<span class='warning'>There is a barely perceptible shimmering of the air above [src].</span>"
+
+/obj/effect/rune/wall/invoke(mob/living/user)
+	density = !density
+	user.visible_message("<span class='warning'>[user] places their hands on [src], and [density ? "the air above it begins to shimmer" : "the shimmer above it fades"].</span>", \
+						 "<span class='warning'>You channel your life energy into [src], [density ? "preventing" : "allowing"] passage above it.</span>")
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		C.apply_damage(2, BRUTE, pick("l_arm", "r_arm"))
+
+
+//Rite of the Unheard Whisper:  Deafens all non-cultists nearby.
+/obj/effect/rune/deafen
+	cultist_name = "Rite of the Unheard Whisper"
+	cultist_desc = "Causes all non-followers nearby to lose their hearing."
+	invocation = "Sti kaliedir!"
+	grammar = "geeri jatkaa karazet"
+
+/obj/effect/rune/deafen/invoke(mob/living/user)
+	visible_message("<span class='warning'>[src] blurs for a moment before fading away.</span>")
+	for(var/mob/living/carbon/C in range(7,src))
+		if(!iscultist(C) && !C.null_rod_check())
+			C << "<span class='warning'>The world around you goes quiet.</span>"
+			C.adjustEarDamage(0,50)
+	qdel(src)
 
 
-/obj/effect/rune/proc/blind()
-	if(istype(src,/obj/effect/rune))
-		var/affected = 0
-		for(var/mob/living/carbon/C in viewers(src))
-			if (iscultist(C))
-				continue
-			var/obj/item/weapon/nullrod/N = locate() in C
-			if(N)
-				continue
+//Rite of the Unseen Glance: Blinds all non-cultists nearby.
+/obj/effect/rune/blind
+	cultist_name = "Rite of the Unseen Glance"
+	cultist_desc = "Causes all non-followers nearby to lose their sight."
+	invocation = "Sti kaliesin!"
+	icon_state = "1"
+	color = rgb(0, 255, 0)
+	grammar = "mgar jatkaa karazet"
+
+/obj/effect/rune/blind/invoke(mob/living/user)
+	visible_message("<span class='warning'>[src] emits a blinding red flash!</span>
```

Comment:
This has always been the case. If it's a problem, I'll add a one-second click delay.